tests/e2e/specFiles/Transactions/receive.spec.ts

```diff
@@ -0,0 +1,135 @@
+import { expect } from 'detox';
+
+import { isBeta, WALLETS_WITH_COINS } from '../../helpers';
+import app from '../../pageObjects';
+
+const DATA_FOR_TRANSACTIONS = {
+  DEFAULT_VALUE: 'Amount',
+  AMOUNT_TO_RECEIVE: '10',
+};
+
+describe('Transactions', () => {
+  beforeEach(async () => {
+    isBeta() && (await app.onboarding.betaVersionScreen.close());
+    await app.developerRoom.tapOnSkipOnboardingButton();
+    await app.navigationBar.changeTab('wallets');
+  });
+
+  describe('Receive', () => {
+    describe('3-Key Vault', () => {
+      describe('@android @iOS @smoke', () => {
+        it('should be possible to see QRCode, wallet address and receive amount for 3-Key Vault wallet', async () => {
+          await app.wallets.createWallet({
```

Comment:
I'd move this part to `beforeEach`. I know there's only one `it()` per `describe()` block, however, I believe it's more readable if the test includes only what's important for the test, without preparation or precondition steps.